doc/website/advanced/configuration-guide.md

```diff
@@ -2,44 +2,64 @@
+for the default values of the constants.
 
 !!! hint
-    With the default values set, the size of `iceoryx_mgmt` is ~64.5 MByte. You can reduce the size by decreasing the values from the table via the CMake options. The current values are printed in the CMake stage when building iceoryx.
+    With the default values set, the size of `iceoryx_mgmt` is ~64.5 MByte. You
+    can reduce the size by decreasing the values from the table via the CMake 
+    options. The current values are printed in the CMake stage when building iceoryx.
 
 Example:
 
 ```bash
 cmake -Bbuild -Hiceoryx_meta -DIOX_MAX_CHUNKS_HELD_PER_SUBSCRIBER_SIMULTANEOUSLY=64
 ```
 
-With that change, the footprint of the management segment is reduced to ~52.7 MBytes. For larger use cases you can increase the value to avoid that samples are dropped on the subscriber side (see also [#615](https://github.com/eclipse-iceoryx/iceoryx/issues/615)).
+With that change, the footprint of the management segment is reduced to ~52.7 MBytes.
+For larger use cases you can increase the value to avoid that samples are dropped
+on the subscriber side (see also [#615](https://github.com/eclipse-iceoryx/iceoryx/issues/615)).
 
 ## :material-memory: Configuring Mempools for RouDi
 
-RouDi supports several shared memory segments with different access rights, to limit the read and write access between different applications. Inside of these segments reside mempools where the user payload data for transfer is stored.
+RouDi supports several shared memory segments with different access rights, to
+limit the read and write access between different applications. Memory pools are
+then managing the those segments and organizing the user payload data required
+for communication.
 
 !!! note
-    Actually only the chunk-payload size is configured and the size of the `ChunkHeader` will be added to the configured size. If a user-header or a user-payload alignment larger than 8 is used, the available size for the user-payload will be smaller than the configured chunk-payload since some space is needed for the other functionality.
-    Please have a look at the `chunk_header.md` design document for a formula how to determine the necessary chunk-payload size with user-header and extended user-payload alignment.
-
-For building RouDi, iceoryx ships a library named `iceoryx_posh_roudi`. This lib gives you an API for compiling your own RouDi application and is part of `iceoryx_posh`.
+    Actually only the chunk-payload size is configured and the size of the
+    `ChunkHeader` will be added to the configured size. If a user-header or a
+    user-payload alignment larger than 8 is used, the available size for the
+    user-payload will be smaller than the configured chunk-payload since some
+    space is needed for the other functionality.
+    Please have a look at the 
+    [chunk_header.md](https://github.com/eclipse-iceoryx/iceoryx/blob/master/doc/design/chunk_header.md)
```

Comment:
```suggestion
    [chunk_header.md](https://github.com/eclipse-iceoryx/iceoryx/blob/v2.0.0/doc/design/chunk_header.md)
```

Reply:
Since it is merged to master I would leave it has it is.